netci.groovy

```diff
@@ -114,6 +113,10 @@ def static calculateBuildCommands(def os, def configuration, def scenario, def i
         if (configuration == 'Debug')
         {
             if (scenario == 'coreclr'){
+                // Run multimodule tests only under CoreCLR mode
+                buildCommands += "tests\\runtest.cmd ${configuration} /multimodule"
```

Comment:
It would be easier to read/understand this if line 110 got changed to always pass `skiptests` and an extra `tests\\runtest.cmd ${configuration}` and `tests\\runtest.cmd ${configuration} /multimodule` command was added if scenario == coreclr.

(Also, line 112 currently still adds a multimodule test pass for the corefx scenario leg.)